desktop/core/src/desktop/js/reactComponents/FileChooser/PathBrowser/PathBrowser.tsx

```diff
@@ -109,9 +125,15 @@ const PathBrowser: React.FC<PathBrowserProps> = ({ breadcrumbs, handleFilePathCh
                         url={item.url}
                         handleFilePathChange={handleFilePathChange}
                         componentType="breadcrumb"
+                        isLastBreadcrumb={index === breadcrumbs.length - 1 ? true : false}
```

Comment:
This is opinionated feedback :-) but I've commented on this before. I don't think breadcrumb should be a setting of an OverflowingItem. I think a breadcrumb should be its own component and internally use an "overflowingItem" of som kind.